utility/src/main/java/org/oppia/android/util/parser/html/PolicyPageTagHandler.kt

```diff
@@ -83,4 +83,14 @@ class PolicyPageTagHandler(
      */
     fun onPolicyPageLinkClicked(policyType: PolicyType)
   }
+
+  override fun getContentDescription(attributes: Attributes): String? {
+    val linkType = attributes.getJsonStringValue("link") ?: return null
+
+    return when (linkType) {
+      TERMS_OF_SERVICE_PAGE -> "Link to "
+      PRIVACY_POLICY_PAGE -> "Link to "
+      else -> null
+    }
+  }
```

Comment:
Could be simplified to:
```suggestion
 override fun getContentDescription(attributes: Attributes): String? {
    return when (attributes.getJsonStringValue("link")) {
        TERMS_OF_SERVICE_PAGE, PRIVACY_POLICY_PAGE -> "Link to "
        else -> null
    }
}
```